_includes/global_fields.md

```diff
@@ -4,6 +4,7 @@ A set of common fields that appear on all resources is included first, for refer
 
 | Name          | Type      | Description
 |-----------    |-----------|-----------|--------------
+|id 			| string    | A string representing a local identifier for the resource on the local system.  It may be a string, integer, guid, etc.  This value MUST be suitable for use in a helper function such as add_tags, add_lists to identify the corresponding resource
```

Comment:
I suspect Josh won't like this... but now that I think about it, we could just get rid of the id property altogether and use resource paths in the add_tags/lists/answers properties, and have the best of both worlds.  After all the resource path basically *is* the id.  I seem to remember we discussed this and Josh thought it was gross. :)